leaflet-geoman.d.ts

```diff
@@ -1776,10 +1776,15 @@ declare module 'leaflet' {
     /**
      * ROTATE MODE MAP EVENT HANDLERS
      */
-    export type GlobalRotateModeToggledEventHandler = (e: {
+    export type BaseEventHandler = (e: { 
+      source: any; 
+      custom?: Record<string, any>; 
+   }) => void;
+
+    export type GlobalRotateModeToggledEventHandler = BaseEventHandler & {
```

Comment:
All the other Events need the BaseEventHandler too.
Additionally I think we should provide a GlobalModeBaseEventHandler, because most of them are the same